src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,17 @@
+.cards {
+  margin: 0 auto;
+  padding: 50px 0;
+  display: flex;
+  flex-wrap: wrap;
+  width: $cards-width;
+}
+
+.card--position {
```

Comment:
It should be `.cards__card` element, not a `card` modifier. So you will have an element which may have margin properties.